### DISCUSSION TITLE
SPEECH {FORMANT} FILTER

### DISCUSSION THREAD

**Discussion by worker-gray** [SynthstromAudible/DelugeFirmware#803] [Ideas]
A new filter type; Formant, could be nested under High Pass.
It wouldn't necessarily have to be 3 parallel band pass filters-- it could be simply 'dial-in the phoneme'.
With some creative step editing, you've got a talking groovebox!
CAVEAT: It may have to work on a 'curve' to handle different key ranges

**Comment by kylenowcrylater** [comment]
+1 I think this would be a rad feature

**Comment by worker-gray** [comment]
the minimal 'traditional' requirements are 3 parallel band pass filters mixed- but there might be a way to fake it. 
The next option would be a vocoder but I already have 2 vocoders and I think programming the deluge to 'talk/sing' would be especially handy and it's probably less taxing on CPU/RAM

**Comment by KyleKirkpatrick** [comment]
Would love to be able to automate the vowel!

**Reply by worker-gray** [reply]
and consonant sounds too-- if possible. full speech synthesis is my hope but I will settle for vowels
